comments/models.py

```diff
@@ -0,0 +1,37 @@
+from django.db import models
+from django.forms import CharField
+
+from films.models import Film
+from users.models import User
+
+class TimeStampModel(models.Model):
+    created_at  = models.DateTimeField(auto_now_add = True)
+    updated_at  = models.DateTimeField(auto_now = True)
+
+    class Meta:
+        abstract = True
+
+class Comment(TimeStampModel):
+    contents   = models.CharField(max_length=500)
+    deleted_at = models.DateTimeField()
+    films      = models.ForeignKey(Film, on_delete=models.CASCADE)
+    users      = models.ForeignKey(User, on_delete=models.CASCADE)
```

Comment:
필드명은 특별한 경우를 제외하고는 단수로 해주세요.
(M2M인 경우 복수로 합니다.)

```suggestion
    content    = models.CharField(max_length=500)
    deleted_at = models.DateTimeField()
    film       = models.ForeignKey(Film, on_delete=models.CASCADE)
    user       = models.ForeignKey(User, on_delete=models.CASCADE)
```